docs/book/how-to/data-artifact-management/handle-data-artifacts/artifacts-naming.md

```diff
@@ -0,0 +1,85 @@
+---
+description: Understand how you can name your ZenML artifacts.
+---
+
+# How Artifact Naming works in ZenML 
```

Comment:
Do you want to remove these pages then? https://docs.zenml.io/how-to/pipeline-development/build-pipelines/dynamically-assign-artifact-names